test/pummel/test-fs-readfile-tostring-fail.js

```diff
@@ -25,13 +25,17 @@ stream.on('error', (err) => { throw err; });
 
 const size = kStringMaxLength / 200;
 const a = Buffer.alloc(size, 'a');
+let expectedSize = 0;
 
 for (let i = 0; i < 201; i++) {
-  stream.write(a);
+  stream.write(a, (err) => { assert.ifError(err); });
```

Comment:
Nit: I think this is not needed because a listener for the `'error'` event is already added on the stream on line 24 but it does not harm.